angularSTEP/src/app/app.module.ts

```diff
@@ -88,7 +86,6 @@ import {UserPageComponent} from './user-page/user-page.component';
     SearchBoxComponent,
     DiscoverPageComponent,
     DiscoverDisplayComponent,
-    ChangeProfileImgComponent,
```

Comment:
This looks like the only place where `ChangeProfileImgComponent` is used - please delete its import!

Reply:
I am a little confused? The change profile image component is used in the user page. It is displayed as a Material Dialog.

Reply:
Ah this seems like a misunderstanding and/or miscommunication on my part: I'm seeing `ChangeProfileImgComponent` being removed from `user-page.component` and that appears to be the only place (other than the import) in all of `user-page.component` which references `ChangeProfileImgComponent`, but does this file need to still reference `ChangeProfileImgComponent` because it's used *elsewhere* in the project? To clarify, I'm not saying delete `ChangeProfileImgComponent` completely from the project - only to remove the now-unused import in this specific component - but IDK if some other file requires that it's still imported here even if unused.

Reply:
Hm I believe I may understand what you are saying? ChangeProfileImgComponent is deleted there because it actually was in the imports twice. It is imported above on line 83. It is still being imported

Reply:
Ah ha! Lol and when I view the full file then I was not seeing the current version at HEAD of main (where it would appear three times in total) but rather the result of this modification (where I see it twice thinking this is deleting one of them). My bad!

Reply:
All good haha! I realized that the second time around too!